2021q3/freebsd-foundation.adoc

```diff
@@ -0,0 +1,169 @@
+link:https://www.FreeBSDfoundation.org/FreeBSD-foundation-partnership-program/[Foundation Partnership Program] URL: link:https://www.FreeBSDfoundation.org/FreeBSD-foundation-partnership-program[https://www.FreeBSDfoundation.org/FreeBSD-foundation-partnership-program] +
+link:https://www.FreeBSDfoundation.org/journal/[FreeBSD Journal] URL: link:https://www.FreeBSDfoundation.org/journal/[https://www.FreeBSDfoundation.org/journal/] + 
+link:https://www.FreeBSDfoundation.org/news-and-events/[Foundation News and Events] URL: link:https://www.FreeBSDfoundation.org/news-and-events/[https://www.FreeBSDfoundation.org/news-and-events/] +
+
+The FreeBSD Foundation is a 501(c)(3) non-profit organization dedicated to
+supporting and promoting the FreeBSD Project and community worldwide. Donations
+from individuals and corporations are used to fund and manage software
+development projects, conferences, and developer summits. We also provide travel
+grants to FreeBSD contributors, purchase and support hardware to improve and
+maintain FreeBSD infrastructure, and provide resources to improve security,
+quality assurance, and release engineering efforts. We publish marketing
+material to promote, educate, and advocate for the FreeBSD Project, facilitate
+collaboration between commercial vendors and FreeBSD developers, and finally,
+represent the FreeBSD Project in executing contracts, license agreements, and
+other legal arrangements that require a recognized legal entity.
+
+Here are some highlights of what we did to help FreeBSD last quarter:
+
+==== Fundraising Efforts
+
+Fundraising last quarter wasn’t as spectacular as we were hoping. But, then
+again, people tend to take vacations during the summer months, which makes it
+that more difficult for our funding requests to go through the management chain
+for approvals.
+
+So far this year we’ve raised $180,000 towards our $2,000,000 spending
+budget. Why do we need so much money? Well, last year we decided to make more
+significant software contributions to FreeBSD.  In order to do that, we had to
+grow our team.  We developed
+link:https://freebsdfoundation.org/blog/technology-roadmap/[a technology
+roadmap] based on input we were receiving from commercial users as well as
+market trends. Based on the roadmap, we identified positions we needed to fill.
+
+This year we've hired three full-time software developers, one full-time ARM
+kernel developer, and one project manager. We also are funding wifi work
+full-time and some other projects to help with FreeBSD on the desktop.  You can
+read about this effort to attract new users and contributors to the Project in
+individual entries elsewhere in this status report.
+
+Our growth wasn’t just in our technology team, but in our advocacy team
+too. Here we hired a marketing coordinator and technical writer to provide more
+educational and informational content. You’ll see in the *Advocacy and
```

Comment:
OK, I'm no longer sure which you settled on, 1 space or 2 after a full stop, but maybe make it consistent?